tests/integration/test_wazuh_db/test_sync_agent_groups_get.py

```diff
@@ -0,0 +1,135 @@
+'''
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
+           Created by Wazuh, Inc. <info@wazuh.com>.
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+type: integration
+brief: Wazuh-db is the daemon in charge of the databases with all the Wazuh persistent information, exposing a socket
+       to receive requests and provide information. The Wazuh core uses list-based databases to store information
+       related to agent keys, and FIM/Rootcheck event data.
+       This test checks the usage of the sync-agent-groups-get command used to allow the cluster getting the
+       information to be synchronized..
+tier: 0
+modules:
+    - wazuh_db
+components:
+    - manager
+daemons:
+    - wazuh-db
+os_platform:
+    - linux
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - CentOS 6
+    - Ubuntu Focal
+    - Ubuntu Bionic
+    - Ubuntu Xenial
+    - Ubuntu Trusty
+    - Debian Buster
+    - Debian Stretch
+    - Debian Jessie
+    - Debian Wheezy
+    - Red Hat 8
+    - Red Hat 7
+    - Red Hat 6
+references:
+    - https://documentation.wazuh.com/current/user-manual/reference/daemons/wazuh-db.html
+tags:
+    - wazuh_db
+'''
+import os
+import time
+import pytest
+import yaml
+import json
+from wazuh_testing.tools import WAZUH_PATH
+from wazuh_testing.wazuh_db import (query_wdb, insert_agent_into_group, clean_agents_from_db,
+                                    clean_groups_from_db, clean_belongs)
+
+
+# Marks
+pytestmark = [pytest.mark.linux, pytest.mark.tier(level=0), pytest.mark.server]
+
+# Configurations
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+messages_file = os.path.join(os.path.join(test_data_path, 'global'), 'sync_agent_groups_get.yaml')
+module_tests = []
+with open(messages_file) as f:
+    module_tests.append((yaml.safe_load(f), messages_file.split('_')[0]))
+
+log_monitor_paths = []
+wdb_path = os.path.join(os.path.join(WAZUH_PATH, 'queue', 'db', 'wdb'))
+receiver_sockets_params = [(wdb_path, 'AF_UNIX', 'TCP')]
+monitored_sockets_params = [('wazuh-db', None, True)]
+receiver_sockets = None  # Set in the fixtures
+agents = ['agent1', 'agent2']
+
+
+# Fixtures
+@pytest.fixture(scope='function')
+def pre_insert_agents():
```

Comment:
Confused name because not only insert agents else also clean all tables.

Reply:
The purpose of this Fixture is to insert agents and assign those agents into a group, the cleans are after a 'yield' and execute after the test to assure that the environment is clean. Maybe I can change the name to pre_insert_agents_into_group but I think that is not necessary to consider the steps inside yield for the name

Reply:
Correct! But not only insert but also remove as you mentioned, so the name should contain the words insert and remove agents. It helps to be representative of the content of the method.